Package.swift

```diff
@@ -0,0 +1,32 @@
+// swift-tools-version: 5.7
+
+import PackageDescription
+
+let package = Package(
+    name: "BitwardenShared",
+    platforms: [
+        .iOS(.v15),
+    ],
+    products: [
+        .library(
+            name: "AuthenticatorSyncShared",
```

Comment:
🤔 I'm not super in love with this name, though I can't think of anything better. Just, if in the future we want to put more things in a shared library between the Password Manager and Authenticator apps, keeping it in the same package makes sense, but the name might not at that point. Probably moot, realistically, but something to think about

Reply:
🤔 I see your point. We could possibly name it like the App Group and go with `BitwardenAuthenticatorShared`? That would simply imply where it's used (in both apps) rather than relating it just to sync.

Alternatively, we could keep this one scoped to just Sync (maybe even name it simply SyncShared) and set a pattern for future packages to be targeted to just one feature? i.e. The main package right now is BitwardenShared, but we are only sharing one product (`AuthenticatorSyncShared`). If you wanted to share Feature X in the future, you could make a FeatureXShared product/target and share via this same mechanism, rather than putting it also in the same product to share. That would add a bit more on the Authenticator side, however, as you'd need to explicitly pull in each product that you're interested in using.

Reply:
Hm. I see merits to both approaches. If we did it by-feature, I'd suggest actually maybe `AuthenticationSyncShared`, so that it's more clearly "this is syncing authentication things".

But also having a broader one that makes it easier in the future for us to just move things from `BitwardenShared` to it in order to share between the two projects... (though personally, I would have called that `BitwardenShared` so I don't know 😂 )

@fedemkr Do you have a strong opinion on whether we want to name more broadly or do a per-feature naming, or something else entirely?

Reply:
Naming is hard, can't think of a really good name for this 🤔 .

I think it depends on the scope (and possible future scope) of this. I would guess that what we're possibly going to share between the two apps would be **auth** data and syncing the **vault**. 
Additionally, I wouldn't put general "helpers" or controls shared between any potential apps as I would add these kind of stuff to an independent package we can reuse amongst all current/future apps without nothing specific of a particular one, e.g. we could also potentially use this future package on the Watch app.

Having said this I'm fine with the **particular naming** for now to denote what the package is about and helps not to include general stuff in there. Furthermore, we can revisit the naming while we include stuff if needed.
I thought about the next ones but not strongly in favor of any of these nor thinking they're better than the original one:
- `AuthenticatorSyncCore`
- `AuthenticatorSyncEngine`
- `AuthenticatorSyncManager`
- `AuthenticatorSyncKit`
- `AuthenticatorSyncFoundation`

Reply:
I'm only a little cautious about naming here because changing the name of a package like this, when it's shared, could be mildly disruptive between the two projects.

I do actually like `Kit` as a suffix; that's something I've used before for this sort of thing, now that I think about it, and it's kind of how Apple likes it. `AuthenticationSyncKit`? `AuthenticatorSyncKit`? Something like that seems like it would be good.

Reply:
Yeah, yesterday reviewed Apple's naming and `Kit` and `Foundation` are usually used as suffixes 😄 . Totally agree let's go with `AuthenticatorSyncKit` 😄 (I wouldn't go for `Authentication` given that if I recall correctly this would also allow the app to send ciphers from one place to the other)

Reply:
Sounds good to me!

Reply:
Sounds good to me as well - I'll migrate to `AuthenticatorSyncKit` 👍

Reply:
@KatherineInCode @fedemkr OK, I think I've got all the names changed over. Latest commit should have everything running with the new name. But do be on the lookout for anything I might have missed.